### PR TITLE
Interrupt channel by mailroom task

### PR DESCRIPTION
**Pull request by norkans7**
requires https://github.com/nyaruka/mailroom/pull/670 first

### CODE REVIEW

File: temba/channels/models.py [hunk 1]
```diff
@@ -905,8 +905,8 @@ def release(self, user, *, trigger_sync: bool = True):
         for sync_event in self.sync_events.all():
             sync_event.release()
 
-        # interrupt any sessions using this channel for calls
-        mailroom.queue_interrupt(self.org, channel=self)
+        # interrupt the channel, any sessions using this channel for calls, fail pending/queued messages and clear courier messages
+        mailroom.queue_interrupt_channel(self.org, channel=self)
```
Comment (rowanseymour) [inline temba/channels/models.py:909]: I wonder if we need this to delay - mailroom instances will have the channel cached for up to 5 seconds after is_active becomes false, and will still be queuing new messages for the channel.. but we queued the task after 5 seconds.. the channel would already be gone...

Comment (norkans7) [reply temba/channels/models.py:909]: we fetch using `models.GetChannelsByID` which does not filter by `is_active = True` I think that will keep working correctly after we switch is_active

Comment (rowanseymour) [reply temba/channels/models.py:909]: yeah but the problem is not loading the channel - the problem is that the channel which has just been deleted in RP, will still be in `GetOrgAssets().Channels()` on all mailroom instances for 5 seconds afterwards.. is there an easy way to have celery delay calling `mailroom.queue_interrupt_channel(self.org, channel=self)` for 5 seconds ?

Comment (rowanseymour) [reply temba/channels/models.py:909]: perfect then let's move `mailroom.queue_interrupt_channel(self.org, channel=self)` into a celery task and do that

Comment (norkans7) [reply temba/channels/models.py]: Added

**Pull request merged**